### PR TITLE
f-form-field@v1.20.0 - Add assistive text and label details

### PR DESCRIPTION
**Pull request by FayeCarter**
### Added
- `labelDetails` and `assistiveText`.
- Styling to placeholder.

image

### CODE REVIEW

**Review by dandel10n — DISMISSED** [review]

**Review by kevinrodrigues — DISMISSED** [review]

File: packages/components/atoms/f-form-field/src/components/FormLabel.vue [hunk 1]
```diff
@@ -103,4 +117,12 @@ input {
         z-index: -1;
     }
 }
+
+.c-formField-label-details {
+    font-weight: $font-weight-regular;
+    color: $form-input-secondaryTextColour;
+    position: absolute;
```
Comment (kevinrodrigues) [inline packages/components/atoms/f-form-field/src/components/FormLabel.vue:124]: Out of interest where does the `position: relative` get set?

Comment (FayeCarter) [reply packages/components/atoms/f-form-field/src/components/FormLabel.vue:124]: I think it's here in the fieldWrapper  https://github.com/justeat/fozzie-components/pull/1134/files/e6584f722d4985237459c75b8bdc16f50bb8d7fd#diff-0c855ee5f708709123fe3e0cd37f1060278c6ead61d3a62a053e5c23b45a03b3R385

File: packages/components/atoms/f-form-field/src/components/FormField.vue [hunk 2]
```diff
@@ -453,4 +474,10 @@ $form-input-iconSize                           : 18px;
     .c-formField-padding--iconTrailing {
         padding-right: $form-input-iconPadding;
     }
+
+    .c-formField-assitiveText {
+        font-weight: $font-weight-regular;
+        color: $color-content-subdued;
```
Comment (dandel10n) [inline packages/components/atoms/f-form-field/src/components/FormField.vue]: Should this be `form-input-secondaryTextColour` as well?

Comment (FayeCarter) [reply packages/components/atoms/f-form-field/src/components/FormField.vue]: 🤦  yurp

**Review by dandel10n — APPROVED** [review]

**Review by kevinrodrigues — APPROVED** [review]

**Pull request merged**